### ISSUE TITLE
Cannot read ING files

### ISSUE THREAD

**Issue by reyezstef** [sandermarechal/jejik-mt940#14]
I suppose this code should be changed in:

Parser/Ing.php

From: 

  public function accept($text)
    {
        return substr($text, 7, 6) === 'INGBNL';
    }

To:

return substr($text, 6, 6) === 'INGBNL';

**Comment by Cyberwizzard** [comment]
I discovered the same issue and when trying to read current MT940 exports from the Dutch ING banking website, I found that the information extracted was incomplete or information ended up in the wrong fields.

I made my own extension which uses this library but provides an updated ING parser which extracts more information. Since I am new to composer, I have uploaded my package to github without adding it to the composer index.

The new ING parser can be found here: https://github.com/Cyberwizzard/mt940/blob/0.3.2/lib/cyberwizzard/MT940/Parser/Ing.php

As I currently do not have access to a recent MT940 bank statement, I can not verify that the extraction of this class into my own library was flawless. This will be tested shortly but feel free to try it out.

My library also adds support for Rabobank CSV formatted bank statements.

Both the ING and Rabobank parser are made using the bank statement format description provided by the respective banks and have been validated using recent bank statement exports.
